tests/unit/style/properties/serialization.rs

```diff
@@ -1115,3 +1115,43 @@ mod shorthand_serialization {
         }
     }
 }
+
+mod longhand_serialization {
+    #[test]
+    fn transform_specified_value_should_serialize_correctly() {
```

Comment:
It would be nice if you also can test these functions separately. It can be hard to determine which function is wrong by looking just one assert in the future.